src/distributions/distributions_jl.jl

```diff
@@ -2,40 +2,8 @@
 # maintained by @zsunberg
 
 import Distributions
-import Distributions: Distribution, UnivariateDistribution, MultivariateDistribution, Matrixvariate
-import Distributions: Categorical, MvNormal
+import Distributions: Categorical
 
-@generated function rand(rng::AbstractRNG, d::Distribution)
-    Core.println("""
-         WARNING: You are using a $d distribution from Distributions.jl. This will work, but simulations will not be repeatable because Distributions.jl does not support non-global random number generators (i.e. there is no `rand(rng::AbstractRNG, d::$d)`)
-
-         For more information, or help Distributions.jl to support this, see https://github.com/JuliaStats/Distributions.jl/issues/436
-
-         To disable this warning, use `rand(rng::AbstractRNG, d::$d) = rand(d)` (you will still not be able to repeat simulations).
-         """)
-    return quote
-        rand(d)
-    end
-end
-
-rand(rng::AbstractRNG, d::UnivariateDistribution) = Distributions.quantile(d, rand(rng)) # fallback
-
-function Base.iterate(d::Categorical)
-    iterate(1:Distributions.ncategories(d))
-end
-function Base.iterate(d::Categorical, state)
-    iterate(1:Distributions.ncategories(d), state)
-end
-
-support(d::Categorical) = 1:Distributions.ncategories(d)
-
-# sampletype(d::UnivariateDistribution) = eltype(d)
-# sampletype(d::MultivariateDistribution) = Vector{eltype(d)}
-# sampletype(d::Distribution{Matrixvariate}) = Matrix{eltype(d)}
-
-# for MvNormal - this should be removed once Distributions.jl PR #597 is in a tagged Distributions.jl release
-rand(rng::AbstractRNG, d::MvNormal) = _rand!(rng, d, Vector{eltype(d)}(undef, length(d)))
-
-function _rand!(rng::AbstractRNG, d::MvNormal, x::VecOrMat)
-    Distributions.add!(Distributions.unwhiten!(d.Σ, randn!(rng, x)), d.μ)
-end
+Base.iterate(d::Categorical) = iterate(1:Distributions.ncategories(d))
+Base.iterate(d::Categorical, state) = iterate(1:Distributions.ncategories(d), state)
+Base.length(d::Categorical) = Distributions.ncategories(d)
```

Comment:
`iterate` is no longer part of the POMDPs.jl interface, and `length` as defined here actually breaks the Distributions.jl concept of `length` being the length of the return type of `rand`, i.e. 1 for Categorical.

That means this whole file should just be deleted.

Reply:
Okay. Did this and squashed commits with previous changes.